pkg/status/status.go

```diff
@@ -51,12 +49,13 @@ const (
 )
 
 func IsUpdatedNUMAResourcesOperator(oldStatus, newStatus *nropv1.NUMAResourcesOperatorStatus) bool {
-	options := []cmp.Option{
-		cmpopts.IgnoreFields(metav1.Condition{}, "LastTransitionTime"),
-		cmpopts.IgnoreFields(metav1.Condition{}, "ObservedGeneration"),
-	}
+	os := *oldStatus.DeepCopy()
+	ns := *newStatus.DeepCopy()
+
+	resetIncomparableFields(&os)
```

Comment:
...and take the address here?

Reply:
because I want to update the copied object without the need to return a value from resetIncomparableFields. It isn't different from passing the original status and constructing a new one from it, update it and return it, do you think the return would be more clear?